guides/common/modules/proc_configuring-dns-nsupdate.adoc

```diff
@@ -0,0 +1,19 @@
+[id="configuring_dns_nsupdate_{context}"]
+= Configuring dns_nsupdate
+
+The _dns_nsupdate_ DNS provider manages DNS records using nsupdate.
+It can be used with any https://www.rfc-editor.org/rfc/rfc2136[RFC2136] compatible nameserver, but by default ISC BIND is installed.
```

Comment:
Interesting: RFC2136 has 1 mention of nameserver and the rest is name server.

Reply:
The rabbit hole goes deeper. https://www.rfc-editor.org/rfc/rfc8499#section-6 suggests to me that it's now preferred to use DNS server